config/environments/development.rb

```diff
@@ -45,4 +45,20 @@
   }
   config.action_controller.default_url_options = { host: 'localhost:3000' }
 
+  +  
++  # These are required to load classes into YAML, eg /test/fixtures/user_tags.yml#L129
++  config.active_record.yaml_column_permitted_classes = [
```

Comment:
Hey @DevPJ9, thanks for working on this.  It seems you added the `+` at the beginning of each line of your changes... the plus sign represented the lines to be added on the issue but the actual code should not include them...kindly recheck. Thanks

Reply:
Okay. Thanks for letting me know. I will do the changes and let you know.

Reply:
Hey @cesswairimu. Changes done. Please review. Thanks.